packages/cli/src/cmds/preview/preview.ts

```diff
@@ -822,4 +849,35 @@ export default class Preview extends YargsCommand {
     }
     this.backgroundTasks = [];
   }
+
+  private async handleDependences(hasBackend: boolean): Promise<[FuncToolChecker, DotnetChecker]> {
+    const cliAdapter = new CLIAdapter(cliEnvCheckerTelemetry, hasBackend);
```

Comment:
There is a build error here. 
```suggestion
    const cliAdapter = new CLIAdapter(hasBackend, cliEnvCheckerTelemetry);
```